api/PaymentItem.json

```diff
@@ -297,10 +297,10 @@
               "version_added": false
             },
             "safari": {
-              "version_added": null
+              "version_added": "11.1"
```

Comment:
I randomly picked this single change to double check. It's been there since https://github.com/WebKit/WebKit/commit/386771f29a48831236a4427dc0854f8d2413e589 so this seems correct in that sense.

But I can't actually find any code in WebKit that *uses* this dictionary member. However, since there's OS integration for this feature, the code for that might not be inside of WebKit. So let's assume this is correct until we get bug reports about it actually not working.